### PR TITLE
refactor: Remove unused code

### PR DESCRIPTION
**Pull request by akash1810**
## What does this change?

First introduced in #336 and later removed in #1157, this change removes some code (and a dependency) for an old custom resource.

## How to test

Tests should continue to pass, (which demonstrates the removed code is no longer used?).

## How can we measure success?

Less code to maintain.

## Have we considered potential risks?

n/a

## Checklist

- [ ] I have listed any breaking changes, along with a migration path [^1]
- [ ] I have updated the documentation as required for the described changes [^2]

[^1]: Consider whether this is something that will mean changes to projects that have already been migrated, or to the CDK CLI tool. If changes are required, consider adding a checklist here and/or linking to related PRs.
[^2]: If you are adding a new construct or pattern, has new documentation been added? If you are amending defaults or changing behaviour, are the existing docs still valid?

### CODE REVIEW

File: tools/integration-test/src/integration-test-stack.test.ts [hunk 1]
```diff
@@ -1,4 +1,3 @@
-import libraryInfo from '@guardian/cdk/lib/constants/library-info';
```
Comment (akash1810) [inline tools/integration-test/src/integration-test-stack.test.ts:1]: Changes in this file are not related to this PR per se, they're made as they're flagged by the linter.

**Review by kenoir — COMMENTED** [review]
Mentioned PRs make it clear this is unused anywhere, so looks good.

**Review by kenoir — APPROVED** [review]
